src/main/java/core/basesyntax/classes/functional/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.classes.functional;
+
+import core.basesyntax.classes.figures.Circle;
+import core.basesyntax.classes.figures.Figure;
+import core.basesyntax.classes.figures.IsoscelesTrapezoid;
+import core.basesyntax.classes.figures.Rectangle;
+import core.basesyntax.classes.figures.RightTriangle;
+import core.basesyntax.classes.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_SIDE_SIZE = 12;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.